### PR TITLE
[Experimental] Kokkos kernels profile

### PR DESCRIPTION
**Pull request by yasahi-hpc**
# About

This PR is the artifact description of the paper entitled “Development of performance portable spline solver for exascale plasma turbulence simulation”. It includes the basic instruction for compilation and experiment workflows. The expected results and the data evaluation method in the paper are also demonstrated.

### CODE REVIEW

File: include/ddc/kernels/splines/splines_linear_problem_band.hpp [hunk 1]
```diff
@@ -155,26 +174,121 @@ class SplinesLinearProblemBand : public SplinesLinearProblem<ExecSpace>
...
+        Kokkos::Profiling::pushRegion(name);
+        if (transpose) {
+            Kokkos::parallel_for(
+                    name,
+                    policy,
+                    KOKKOS_CLASS_LAMBDA(const int i) {
+                        auto sub_b1 = Kokkos::subview(b1, Kokkos::ALL, i);
+                        auto sub_b2 = Kokkos::subview(b2, Kokkos::ALL, i);
+
+                        KokkosBatched::SerialGemm<
```
Comment (blegouix) [inline include/ddc/kernels/splines/splines_linear_problem_band.hpp:238]: You can use Gemv right ? I guess they have some optimizations compared to Gemm

Comment (yasahi-hpc) [reply include/ddc/kernels/splines/splines_linear_problem_band.hpp:238]: I think so. I guess it is also the case for PR #480

Comment (blegouix) [reply include/ddc/kernels/splines/splines_linear_problem_band.hpp:238]: I don't think so, in #480 this is matrix-matrix multiplication.

Comment (yasahi-hpc) [reply include/ddc/kernels/splines/splines_linear_problem_band.hpp:238]: You are right. It can be Gemv since we operate on subview. I will fix this

**Comment by tpadioleau** [comment]
Closing, superseded by the fork https://github.com/yasahi-hpc/ddc

**Pull request closed without merging**